### PR TITLE
[TW-921] Doc file-less API definitions

### PR DESCRIPTION
**Pull request by mark-anthony-postman**

### CODE REVIEW

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 1]
```diff
@@ -138,9 +142,9 @@ To add a file to a folder, select the more actions icon <img alt="More actions i
...
 
-> If you delete the last remaining file in an API definition, the API will no longer have a definition unless you create it again. The schema documentation and any information in the file's info pane including the definition ID will be lost. Other components of the API will remain intact. For example, collections in the API won't be deleted. You can't restore a definition after it's deleted.
+> **About definition IDs** When you add a definition to an API, Postman assigns a definition ID to the API. You can view the definition ID by opening an API and selecting the information icon <img alt="Information icon" src="https://assets.postman.com/postman-docs/icon-information-v9-5.jpg#icon" width="16px"> in the right sidebar. The definition ID acts as a container for all the definition files in the API. If you delete all the definition files, the definition ID itself isn't deleted. If you then add a new definition file, the definition ID remains the same as before.
```
Comment (MatthewPinsker-postman) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:147]: This is minor, but what do you think about using a character to separate "**About definition IDs**" from the rest of the note? Something like, "**About definition IDs:** When you add a definition to..." or "**About definition IDs.** When you add a definition to..."

My thinking is that the separating character helps clarify that **About definition IDs** is an intro/title, and not part of the first sentence.

**Review by MatthewPinsker-postman — APPROVED** [review]
Looks good!

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md (hunk 1 above)
Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md]: Thanks for spotting that Matthew! My intention was to have a period there, but my fingers forgot to type it 😅

**Review by petewitcher-postman — APPROVED** [review]
LGTM!

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 2]
```diff
@@ -138,9 +142,9 @@ To add a file to a folder, select the more actions icon <img alt="More actions i
 
 ### Deleting files and folders
 
-To delete a file or folder, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the item and select **Delete**.
+To delete a definition file or folder, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the item and select **Delete**. Deleting a file or folder doesn't affect other elements added to the API, such as collections. You can't restore a definition file or folder after it's deleted.
```
Comment (raghumurthy-jayandran) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:145]: With the file-less definition you will now be able to restore a deleted file.

**Review by raghumurthy-jayandran — CHANGES_REQUESTED** [review]

File: src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md [hunk 3]
```diff
@@ -138,9 +142,11 @@ To add a file to a folder, select the more actions icon <img alt="More actions i
 
 ### Deleting files and folders
 
-To delete a file or folder, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the item and select **Delete**.
+To delete a definition file or folder, select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> next to the item and select **Delete**. Deleting a file or folder doesn't affect other elements added to the API, such as collections.
+
+You can restore a deleted definition file use the [Changelog](/docs/designing-and-developing-your-api/managing-apis/#using-the-changelog). Select the changelog icon <img alt="Changelog icon" src="https://assets.postman.com/postman-docs/icon-changelog-v9.jpg#icon" width="18px"> in the right sidebar, then select **Restore** below the definition file you want to restore.
```
Comment (raghumurthy-jayandran) [inline src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:147]: @mark-anthony-postman Shouldn't the sentence  say `You can restore a deleted definition file using [Changelog]`

Comment (mark-anthony-postman) [reply src/pages/docs/designing-and-developing-your-api/developing-an-api/defining-an-api.md:147]: Yes, it should be "using the [Changelog]". Thanks so much for catching that mistake!

**Review by raghumurthy-jayandran — APPROVED** [review]

**Pull request merged**